### PR TITLE
the point size, time window and curve style can be definded from python

### PR DESCRIPTION
**Pull request by jbcolme**

### CODE REVIEW

File: src/signal_scope/mainwindow.h [hunk 1]
```diff
@@ -47,6 +47,9 @@ public slots:
   void onChooseBackgroundColor();
   void onChooseHistoryLength();
   void onChoosePointSize();
+  void onPointSizeChanged(int size);
+  void onCurveStyleChanged(QString style);
+  void onTimeWindowChange(double timeWindow);
```
Comment (patmarion) [inline src/signal_scope/mainwindow.h]: rename to "onTimeWindowChanged"

File: src/signal_scope/signalScopeSetup.py [hunk 2]
```diff
@@ -147,12 +147,23 @@ def decodeMessageFunction(messageBytes):
 msg = LookupHelper()
 tNow = LocalTimeHelper()
 
-def addPlot(timeWindow=None, yLimits=None):
+def formatOptions(pointSize=None, timeWindow=None, curveStyle=None):
```
Comment (patmarion) [inline src/signal_scope/signalScopeSetup.py]: rename to "setFormatOptions"

**Comment by patmarion** [comment]
Looks great!  I like it.

I just made two in line comments about minor function name changes, thanks!

File: examples/util_example.py [hunk 3]
```diff
@@ -12,21 +12,25 @@ def makePlots(joints):
...
 
 
     # velocity plot
-    addPlot(timeWindow=30, yLimits=[-2, 2])
+    addPlot(yLimits=[-2, 2])
     for joint in joints:
         addSignal('EST_ROBOT_STATE', msg.utime, msg.joint_velocity[jn[joint]])
         addSignal('ATLAS_COMMAND', msg.utime, msg.velocity[jns[joint]])
+
+# you can select the point size, time window and curvstyle for all plots with the function formatOptions
```
Comment (patmarion) [inline examples/util_example.py]: small typo "curve style"

**Comment by mauricefallon** [comment]
Great to see the contributions from HyQ lab. thanks!

**Comment by jbcolme** [comment]
Done! 
Thanks for going over the code .
Cheers.

**Pull request merged**